### PR TITLE
Add ant-design-mobile@0.9.11 /w npm autoupdate

### LINKED ISSUE #9689: [Request] ant-design-mobile
**Library name:ant-design/ant-design-mobile** 
**Git repository url:https://github.com/ant-design/ant-design-mobile.git**
**npm package url(optional):https://www.npmjs.com/package/antd-mobile** 
**License(s):MIT**
**Official homepage:http://mobile.ant.design/**
**Wanna say something? Leave message here:A configurable Mobile UI specification and React-based implementation.**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #9689 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ant-design/ant-design-mobile.git
 * Official website (optional, not the repository): http://mobile.ant.design/
 * NPM package url (optional): https://www.npmjs.com/package/antd-mobile
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 858
   - Count of watchers: 78
   - Count of forks: 116
 * NPM download stats (optional):
   - Downloads in the last day: 63
   - Downloads in the last week: 1449
   - Downloads in the last month: 4185

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9689, cc @ant-design

### CODE REVIEW

File: ajax/libs/antd-mobile/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "type": "git",
+    "url": "http://github.com/ant-design/ant-design-mobile"
+  },
+  "license": "MIT",
+  "npmName": "antd-mobile",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/antd-mobile/package.json]: `"**/!(*.less)"` would be better since we don't host the "less" file type.

**Comment by PeterDaveHello** [comment]
@extend1994 we should fix it but not close and open a new one

**Comment by extend1994** [comment]
I thought that I should remove `.less` file in `/dist` and recreate a branch also named as antd-mobile, then type `git push myorigin antd-mobile` at last.
But the issue closed automatically.
Was I wrong with my thought?

**Comment by extend1994** [comment]
@PeterDaveHello I found that I had forgotten to type `add` and `commit` before `push`, I just fixed it.

**Pull request merged**